packages/design-system/src/components/GlobalStyle/GlobalStyle.ts

```diff
@@ -16,9 +16,11 @@
 // =============================================================================
 import { createGlobalStyle } from "styled-components";
 import { rem } from "polished";
-import { palette, spacing } from "../../styles";
-import "@fontsource/public-sans";
+import { palette, spacing, typography } from "../../styles";
+import "@fontsource/public-sans/400.css";
 import "@fontsource/public-sans/500.css";
+import "@fontsource/public-sans/600.css";
+import "@fontsource/public-sans/700.css";
 import "@fontsource/libre-franklin";
 import "@fontsource/libre-franklin/500.css";
```

Comment:
Can these be dropped?

Reply:
oh yes I believe so, that was an oversight. Good catch!